### DISCUSSION TITLE
Dropbox link disabled

### DISCUSSION THREAD

**Discussion by xGraveborn** [kinkofer/FightClub5eXML#163] [Q&A]
Hey I was trying to download the finished file to upload into FightClub but the link in your read me is disabled. Do you have another way to get the file?

**Comment by OkayestTurtle** [comment]
I also was trying to do this but the link has been disabled. Is there a new link or how can I receive access to it?

**Comment by asenci** [comment]
It's a known issue, unfortunately: https://github.com/kinkofer/FightClub5eXML/issues/157#issuecomment-877484276
